bgpd/bgp_mplsvpn.h

```diff
@@ -107,11 +107,8 @@ static inline int vpn_leak_to_vpn_active(struct bgp *bgp_vrf, afi_t afi,
 }
```

Comment:
In this function (vpn_leak_to_vpn_active), there is no check on instance type. Of course, that is not changed in this commit, but I happened to compare with its counterpart below (vpn_leak_from_vpn_active) which has the check.

Reply:
Added check.